packages/amplify-e2e-tests/src/plugin/index.ts

```diff
@@ -1,34 +1,31 @@
-export * from './new-plugin';
+export * from './newPlugin';
 export * from './verifyPluginStructure';
+export * from './pluginConfigure';
 
 import { nspawn as spawn } from 'amplify-e2e-core';
 import { getCLIPath } from '../utils';
 
 export function help(cwd: string) {
   return new Promise((resolve, reject) => {
-    spawn(getCLIPath(), ['plugin', 'help'], { cwd, stripColors: true })
-      .wait(/.*/)
-      .run(function(err: Error) {
-        if (!err) {
-          resolve();
-        } else {
-          reject(err);
-        }
-      });
+    spawn(getCLIPath(), ['plugin', 'help'], { cwd, stripColors: true }).run(function(err: Error) {
+      if (!err) {
+        resolve();
+      } else {
+        reject(err);
+      }
+    });
   });
 }
 
 export function scan(cwd: string) {
```

Comment:
Why isn't this e2e test just checking the full output? I think we need better tests here. This is the exact current output for me:

```amplify plugin scan```
Scanning for plugins...
Plugin scan successful
Scanning for plugins...
Plugin scan successful

Reply:
This method simulates the command line UX workflow for the command. The command itself does not have any interaction with the user after it starts. It either runs successfully or it throws error.